feed-rs/Cargo.toml

```diff
@@ -22,7 +22,7 @@ readme = "README.md"
 travis-ci = { repository = "feed-rs/feed-rs", branch = "master" }
 
 [dependencies]
-chrono = { version = "0.4" }
+chrono = { version = "0.4.1", default-features = false }
 lazy_static = "1.4"
```

Comment:
Hi @malyn , thanks very much for the patch.

I'm not much of an expert when it comes to cargo and dependency rules ... just curious why we wouldn't go straight to `0.4.22` ?

Reply:
Hi @markpritchard, you're welcome!

My preference/recommendation would be to go to 0.4.22 since that version fixes a lot of issues in previous versions, eliminates the RUSTSEC vulnerability, and also fixes some issues that cropped up in 0.4.20 and 0.4.21.

The only downside to doing that is that every user of feed-rs would also now have to upgrade to at least chrono 0.4.22. Again, I think that's okay, but feed-rs was only requiring 0.4 until this PR and I wasn't sure what your policy was on dependency upgrades.

Happy to modify the PR to switch to 0.4.22 (which will also need a slight adjustment to the `default-features` and `features` attributes) if you would prefer to move to the latest version.

Reply:
Hi @malyn , I bumped it to 0.4.23 following your comment above. Thanks for the PR!